README.md

```diff
@@ -53,8 +53,7 @@ Readable? Efficient? Can your Python/Ruby/V8 do better?
 Status
 ------
 
-**Lua Fun** is in an early alpha stage. The library fully [documented]
-[Documentation] and covered with unit tests.
+**Lua Fun** is in an early alpha stage. The library fully [Documentation] and covered with unit tests.
```

Comment:
`fully [Documentation]` → `fully [documented][Documentation]`

IOW, the fix is to get rid from improper newline.